src/animation.jl

```diff
@@ -275,6 +289,11 @@ Example:
     push!(p, 1, sin(x))
   end
 ```
+This macro supports additional parameters, that may be added after the main loop body.
+- Add `fps=n` with positive Integer n, to specify the desired frames per second.
+- Add `every n` with positive Integer n, to take only one frame every nth iteration.
+- Add `when <cond>` where `<cond>` is an Expression resulting in a Boolean, to take a 
+    frame only when `<cond>` returns `true`. Is incompatible with `every`.
 """
 macro apng(forloop::Expr, args...)
```

Comment:
I tried this, with 
```julia
let moreDocumentation = """This macro supports ... """
    @doc string(@doc @gif)*moreDocumentation :(@gif)
    @doc string(@doc @apng)*moreDocumentation :(@apng)
    @doc string(@doc @animate)*moreDocumentation :(@animate)
end
```
but instead of extending the documentation it added a whole new one. It was not very pretty.